src/test/java/org/springframework/cloud/dataflow/acceptance/test/AbstractTaskTests.java

```diff
@@ -234,19 +236,31 @@ protected void registerTasks() {
 	 */
 	public void afterPropertiesSet() {
 		if (restTemplate == null) {
-			try {
-				DataFlowTemplate dataFlowOperationsTemplate = new DataFlowTemplate(
-						new URI(configurationProperties.getServerUri()));
-				taskOperations = dataFlowOperationsTemplate.taskOperations();
-				schedulerOperations = dataFlowOperationsTemplate.schedulerOperations();
-				appRegistryOperations = dataFlowOperationsTemplate.appRegistryOperations();
-				this.jobOperations = dataFlowOperationsTemplate.jobOperations();
-			}
-			catch (URISyntaxException uriException) {
-				throw new IllegalStateException(uriException);
-			}
-			restTemplate = new RestTemplate();
+			configureRestTemplate();
+			DataFlowTemplate dataFlowOperationsTemplate = DataFlowTemplateConfigurer
+						.create(configurationProperties.getServerUri()).configure();
+			taskOperations = dataFlowOperationsTemplate.taskOperations();
+			schedulerOperations = dataFlowOperationsTemplate.schedulerOperations();
+			appRegistryOperations = dataFlowOperationsTemplate.appRegistryOperations();
+			jobOperations = dataFlowOperationsTemplate.jobOperations();
+			restTemplate = dataFlowOperationsTemplate.getRestTemplate();
+		}
+	}
+
+	/**
+	 * Configure RestTemplate to skip ssl validation
+	 */
+	protected void configureRestTemplate() {
+		HttpClientConfigurer httpClientConfigurer = null;
+		try {
+			httpClientConfigurer = HttpClientConfigurer.create(
+					new URI(configurationProperties.getServerUri()))
+					.skipTlsCertificateVerification(true);
+		} catch (URISyntaxException e) {
```

Comment:
So we agreed we need to have the skipSSL?   Just curious.

Reply:
I think we do, and still don't have an explanation for how it can work with out it.  @ilayaperumalg ?